### ISSUE TITLE
Multi-currency transactions are ignored by QIF

### ISSUE THREAD

**Issue by codinguser** [codinguser/gnucash-android#571]
#### Expected behaviour

All transactions should be included in the file, even if they are only using the converted amount
#### Actual behaviour

Multicurrency transactions are completely ignored
#### Steps to reproduce the behaviour
1. Create multi-currency transactions
2. Export QIF from the app
#### Software specifications
- GnuCash Android version: 2.1.1
- Android version: Any
- Device type: Any

**Comment by codinguser** [comment]
@rivaldi8 would it be possible for you to look into this?
I will move it to milestone 2.1.4 so that I can go ahead and release 2.1.3 and all the fixes already in it.

**Comment by rivaldi8** [comment]
Ok! I'll look into it.

**Comment by rivaldi8** [comment]
I've sent a pull request, hopefully fixing the issue.

Now the multi-currency transactions are exported, but I'm not sure if they're exported in the expected way. In my tests I got two cases.

* Entering an amount of 20 in Local, 15 in converted amount (Foreign)
```
!Account
NExpenses:Local
^
!Type:Cash
D2016/11/10
MDescription
SExpenses:Foreign
$15,00
^
```
* Entering an amount of 10 into Local:
```
!Account
NAssets:Foreign
^
!Type:Oth A
D2016/11/10
MDescription
SExpenses:Local
$-10,00
```

**Comment by rivaldi8** [comment]
Fixed in hotfix/patches (f9b3a30).
